drivers/cc110x/include/cc110x_communication.h

```diff
@@ -205,6 +205,20 @@ uint8_t cc110x_status(cc110x_t *dev);
  */
 int cc110x_power_on(cc110x_t *dev);
 
+/**
+ * @brief   Brings the transceiver into IDLE mode from RX.
+ *
+ * This is needed to get the state machine into IDLE before going to SLEEP.
+ *
+ */
+void cc110x_wakeup_from_rx(cc110x_t *dev);
```

Comment:
Does this need to be a public function?

Reply:
I think it might be used for other purposes, e.g. if the user needs to ensure the IDLE state after a RX transition.

Reply:
For this the name and description are rather narrow.

I see it basically only does

```C
    cc110x_cmd(dev, CC110X_STROBE_IDLE);
    dev->state = CC110X_STATE_IDLE;
```

which is used elsewhere in the code.

Reply:
Ok, I think I can delete it and bring it into the power off one.

Reply:
just make it `static`.